src/kendo.numerictextbox.js

```diff
@@ -501,12 +513,27 @@ var __meta__ = { // jshint ignore:line
 
                 e.preventDefault();
             } else if ((min !== null && min >= 0 && value.charAt(0) === "-") || !isValid) {
+                that._blinkInvalidState();
                 e.preventDefault();
             }
 
             that._key = 0;
         },
 
+        _blinkInvalidState: function () {
+            var that = this,
+                wrapper = that._inputWrapper.addClass(INVALID);
+
+            that._validationIcon.show();
+            clearTimeout(that._blinkTimeout);
+
+            that._blinkTimeout = setTimeout(function(){

```

Comment:
The code inside the timeout function can be extracted to a separate method, such as `_removeInvalidState` (make sure to call it with `$.proxy` to be sure of the this object).